packages/media/src/types/audioResource.ts

```diff
@@ -17,11 +17,13 @@
 /**
  * Internal dependencies
  */
-import type { ResourceType } from './resourceType';
 import type { Resource } from './resource';
 
-export interface AudioResource extends Resource {
-  type: ResourceType.Audio;
+export interface AudioResource
+  extends Pick<
+    Resource,
+    'type' | 'id' | 'src' | 'mimeType' | 'isExternal' | 'isPlaceholder'
+  > {
```

Comment:
This type is not 100% accurate because `AudioResource` doesn't actually use any of these fields in the editor. It doesn't really extend `Resource`. Something to fix in the future.